kornia/augmentation/container/augment.py

```diff
@@ -25,6 +28,13 @@ class AugmentationSequential(ImageSequential):
             applied to each. If None, it will not overwrite the function-wise settings. Default: None.
         keepdim (bool, optional): whether to keep the output shape the same as input (True) or broadcast it
             to the batch form (False). If None, it will not overwrite the function-wise settings. Default: None.
+        random_apply(int, (int, int), optional): randomly select a sublist (order agnostic) of args to
```

Comment:
```suggestion
        random_apply: randomly select a sublist (order agnostic) of args to
```
we won't need to add typing by hand in docs anymore neither defaults